assets/js/admin/marketplace-suggestions.js

```diff
@@ -170,6 +170,26 @@
 				container.appendChild( body );
 			}
 
+			// Conditionally add in a Manage suggestions link to product edit
+			// metabox footer (based on suggestion slug).
+			var slugsWithManage = [
+				'product-edit-empty-footer-browse-all',
```

Comment:
Note that I've added another alias for this suggestion, as the original one was misleading / confusing. I'm renaming it in the json data, but I've got both here. In future we can remove `product-edit-empty-footer-browse-all`.